### ISSUE TITLE
color status bar in settings toolbar explorer

### ISSUE THREAD

**Issue by qzmtch** [Open-Shell/Open-Shell-Menu#614]
you can in future versions add color selection status bar? it's just that if you install various third-party themes, users usually install oldnewexplorer and after it, the big status bar is removed at the bottom through group policies, and I wanted to return it for myself through the open shell, but it can't change its color, it looks [like this](https://i.imgur.com/3qhFYZo.png) if in qttabbar you can still change the color, then in the status bar you cannot

**Comment by bonzibudd** [comment]
This has been previously discussed in https://github.com/Open-Shell/Open-Shell-Menu/issues/361

**Comment by qzmtch** [comment]
> This has been previously discussed in #361

yes, but, the developers in that thread did not answer, they could just write if they would add a color setting or not
